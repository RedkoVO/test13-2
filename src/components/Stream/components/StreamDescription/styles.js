import Heart from '../../../../assets/images/svg/heart.svg'
import Reply from '../../../../assets/images/svg/reply.svg'

export default () => ({
  root: {
    display: 'flex',
    justifyContent: 'space-between',
    paddingTop: '30px'
  },
  streamDescriptionLeft: {
    marginRight: '50px',
    width: '650px'
  },
  streamDescriptionText: {
    color: '#fff',
    fontSize: '18px',
    fontWeight: 'lighter'
  },
  streamHashTags: {
    display: 'flex',
    alignItems: 'center',
    paddingTop: '6px',
    color: '#fff',
    fontSize: '28px',
    fontWeight: 'bold'
  },
  streamAddHashTags: {
    marginLeft: '10px',
    width: '17px',
    height: '17px',
    lineHeight: '15px',
    fontSize: '19px',
    textAlign: 'center',
    borderRadius: '5px',
    backgroundColor: 'rgba(0, 0, 0, 0.37)'
  },
  streamDescriptionRight: {
    width: '150px',
    color: '#fff',
    fontSize: '13px',
    fontWeight: 'bold'
  },
  streamAddFavorites: {
    position: 'relative',
    marginBottom: '8px',
    width: '145px',
    padding: '6px 0',
    paddingLeft: '5px',
    fontSize: '12px',
    fontWeight: 'bold',
    textAlign: 'center',
    backgroundColor: 'rgba(255, 255, 255, 0.15)',
    borderRadius: '10px',

    '&:before': {
      position: 'absolute',
      content: `""`,
      top: '7px',
      left: '11px',
      width: '11px',
      height: '11px',
      background: `url(${Heart})`,
      backgroundSize: '11px',
      backgroundRepeat: 'no-repeat',
      backgroundPosition: '0 0'
    }
  },
  streamShareSocial: {
    position: 'relative',
    marginBottom: '8px',
    width: '145px',
    padding: '6px 0',
    paddingLeft: '5px',
    fontSize: '12px',
    fontWeight: 'bold',
    textAlign: 'center',
    backgroundColor: 'rgba(255, 255, 255, 0.15)',
    borderRadius: '10px',

    '&:before': {
      position: 'absolute',
      content: `""`,
      top: '7px',
      left: '11px',
      width: '11px',
      height: '11px',
      background: `url(${Reply})`,
      backgroundSize: '11px',
      backgroundRepeat: 'no-repeat',
      backgroundPosition: '0 0'
    }
  }
})
