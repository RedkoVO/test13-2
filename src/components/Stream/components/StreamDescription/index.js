import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import styles from './styles'

const StreamDescription = ({ classes }) => (
  <div className={classes.root}>
    <div className={classes.streamDescriptionLeft}>
      <div className={classes.streamDescriptionText}>
        Integer tempor semper sapien sit amet bibendum. Etiam ornare tellus at
        sapien vehicula, quis scelerisque turpis rokaste egestas quis botona.
      </div>
      <div className={classes.streamHashTags}>
        #ROLLED #BESTCRYPTO #ETH{' '}
        <div className={classes.streamAddHashTags}>+</div>
      </div>
    </div>
    <div className={classes.streamDescriptionRight}>
      <div className={classes.streamAddFavorites}>Add to favorites</div>
      <div className={classes.streamShareSocial}>Share in social</div>
    </div>
  </div>
)

StreamDescription.propTypes = {
  classes: PropTypes.object
}

export default withStyles(styles)(StreamDescription)
