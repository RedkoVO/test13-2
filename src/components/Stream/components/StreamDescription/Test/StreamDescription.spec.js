import * as React from 'react'
import renderer from 'react-test-renderer'

import StreamDescription from '../'

it('renders correctly', () => {
  const tree = renderer.create(<StreamDescription />).toJSON()
  expect(tree).toMatchSnapshot()
})
