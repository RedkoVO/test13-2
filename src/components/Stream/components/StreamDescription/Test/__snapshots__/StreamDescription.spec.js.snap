// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="StreamDescription-root-1"
>
  <div
    className="StreamDescription-streamDescriptionLeft-2"
  >
    <div
      className="StreamDescription-streamDescriptionText-3"
    >
      Integer tempor semper sapien sit amet bibendum. Etiam ornare tellus at sapien vehicula, quis scelerisque turpis rokaste egestas quis botona.
    </div>
    <div
      className="StreamDescription-streamHashTags-4"
    >
      #ROLLED #BESTCRYPTO #ETH
       
      <div
        className="StreamDescription-streamAddHashTags-5"
      >
        +
      </div>
    </div>
  </div>
  <div
    className="StreamDescription-streamDescriptionRight-6"
  >
    <div
      className="StreamDescription-streamAddFavorites-7"
    >
      Add to favorites
    </div>
    <div
      className="StreamDescription-streamShareSocial-8"
    >
      Share in social
    </div>
  </div>
</div>
`;
